charts/activemq-artemis/values.yaml

```diff
@@ -11,8 +11,12 @@ imageTag: latest
 
 ## Create a database user
 ##
-artemisUser: artemis
-artemisPassword: simetraehcapa
+artemis:
```

Comment:
We are already talking about activemq-artemis. Maybe authentication is a better subject?